src/Razor/src/Microsoft.VisualStudio.RazorExtension/language-configuration.json

```diff
@@ -0,0 +1,28 @@
+﻿{
+  "comments": {
+    "blockComment": [ "@*", "*@" ]
+  },
+  "brackets": [
+    ["<!--", "-->"],
+    ["<", ">"],
+    ["{", "}"],
+    ["(", ")"]
+  ],
+  "autoClosingPairs": [
+    { "open": "{", "close": "}"},
+    { "open": "[", "close": "]"},
+    { "open": "(", "close": ")" },
+    { "open": "'", "close": "'" },
+    { "open": "\"", "close": "\"" },
+    { "open": "@*", "close": "*@" }
+  ],
+  "surroundingPairs": [
+    { "open": "'", "close": "'" },
+    { "open": "\"", "close": "\"" },
+    { "open": "<", "close": ">" }
+  ],
+  "indentationRules": {
+    "increaseIndentPattern": "(?:<(?!(?:area|base|br|col|embed|hr|img|input|keygen|link|menuitem|meta|param|source|track|wbr))(\\w[\\w\\d]*)([^/>]*(?!/)>)[^<]*$)|(?:\\{\\s*$)",
```

Comment:
Where did these come from?

Reply:
Our [RazorLanguageConfiguration](https://github.com/dotnet/aspnetcore-tooling/blob/92f4d1e581e73fef94c37f88b1d664b394137173/src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorLanguageConfiguration.ts#L35) handler in VSCode. Only difference is there is no `OnEnterRules` for VS so I had to repurpose it + add the support for curly braces